lemur/certificates/verify.py

```diff
@@ -69,7 +69,14 @@ def ocsp_verify(cert, cert_path, issuer_chain_path):
         raise Exception(f"OCSP lookup timed out: {url}, certificate serial number {cert.serial_number:02X}")
 
     p_message = message.decode("utf-8")
-    if "error" in p_message or "Error" in p_message:
+
+    if "unauthorized" in p_message:
+        # indicates the OCSP server does not know this certificate
+        metrics.send("check_revocation_ocsp_verify", "counter", 1, metric_tags={"status": "unauthorized", "url": url})
+        current_app.logger.warning(f"OCSP unauthorized error:{url}")
```

Comment:
Does it make sense to add serial number to this log as well? It doesn't seem very useful without that.

Reply:
Oops, let me add it

Reply:
Actually the caller logs return exception as warning, so I'll remove this log. For other conditions too we do the same